contributing/getting-involved/en.md

```diff
@@ -51,15 +51,19 @@ Our Help Center improves when the documentation improves. If there are articles
 
 ## Packaging
 
-There may be software you wish to have on your system that is currently not available in the repository. While you are more than welcome to submit a bug and wait for someone else to package the software you want, you can also improve 
+There may be software you wish to have on your system that is currently not available in the repository. While you are more than welcome to submit a bug and wait for someone else to package the software you want, you can also improve
 and contribute to the community by packaging software. If you're interested in learning to package software, feel free to visit our documentation [here](/articles/packaging/).
 
 ## Report Bugs
 
 We're always looking to improve our systems, especially when they're not functioning as expected. By reporting bugs, you improve the system not only for you, but for all Solus Project users.
 
 - [Developer Portal](https://dev.getsol.us)
+Most bug reports should be filed here.
+
+- [Solus Github Repos](https://github.com/getsolus)
+Certain parts of Solus, such as the Budgie Desktop, have repos on github. Check here to see if the system in question is listed. If so, file the bug in the appropriate repo.
```

Comment:
Since Budgie is actually via the solus-project org, can we please have an item above Development Portal for Budgie Desktop issues.